pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -1,6 +1,8 @@
 { stdenv
 , lib
-, cudatoolkit
+, zlib
+, autoPatchelfHook
+, cudaPackages
```

Comment:
Because this package is part of cudaPackages I think it is better to pass in the individual deps instead of the whole set.

Reply:
Indeed. I should maybe remove `passthru.cudaPackages` then - downstream derivations should on the opposite query `cudaPackages`, not `cudnn` directly

Reply:
That's a good one...